.github/workflows/build-docker.yaml

```diff
@@ -0,0 +1,60 @@
+name: Build Docker Images
+
+on: [push, pull_request]
+env:
+  TARGET_PLATFORMS: linux/amd64,linux/arm64
+
+jobs: 
+  build-agent-docker:
+    strategy:
+      matrix:
+        platform:
+          - linux/amd64
```

Comment:
What is the advantage of building an image per architecture versus one multi arch image?

Reply:
The output artefact in ghcr.io gets updated with both architectures from both jobs it doesn't make multiple artefacts.

Also I noticed it was really slow to compile when doing parallel buildx building so I figured an action job per arch and image combo would be faster if only slightly.